.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 AllCops:
   TargetRubyVersion: 2.2
-
+Metrics/BlockLength:
```

Comment:
I don't quite understand why we're making this change?

Reply:
We had a discussion about this in #1224 but ultimately decided we don't need it.  This commit needs to be dropped form this PR (and any others it's been introduced in).

Reply:
Dropped 👍